tests/system/src/test/java/test/com/sun/javafx/application/InitializeJavaFXLaunch1Test.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.  Oracle designates this
+ * particular file as subject to the "Classpath" exception as provided
+ * by Oracle in the LICENSE file that accompanied this code.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+package test.com.sun.javafx.application;
+
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class InitializeJavaFXLaunch1Test extends InitializeJavaFXLaunchBase {
+
+    @BeforeClass
+    public static void initialize() throws Exception {
+        System.out.println("Calling Startup!");
+        InitializeJavaFXLaunchBase.initializeApplicationLaunch();
+        System.out.println("Called Startup!");
+    }
+
+    @Test (timeout = 15000)
+    public void testStartupThenLaunchInFX() throws Exception {
```

Comment:
Would something like `testLaunchThenLaunchInFX` be a better name? As it is, it uses the same name as the test that really does use `Platform.startup` which I found confusing.

Reply:
Yes, I've fixed the method names!